crates/flox/src/commands/environment.rs

```diff
@@ -482,24 +483,25 @@ fn environment_description(environment: &ConcreteEnvironment) -> Result<String,
 /// Generate a description for an environment that has not yet been opened.
 ///
 /// TODO: we should share this implementation with environment_description().
-/// We probably need an UnopenedEnvironment or LightweightEnvironment or
-/// EnvironmentDescriptor that represents a partially opened environment.
 pub fn hacky_environment_description(
```

Comment:
quabble: this will need to change once move when we have Remote envs that have no (exposed) anchor on the user's machine